File: Datasets/HumanEval/HumanEval_Java/56_P13.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class CorrectBracketing {
    /**
     * brackets is a string of "<" and ">".
     * return True if every opening bracket has a corresponding closing bracket.

     * >>> correct_bracketing("<")
     * False
     * >>> correct_bracketing("<>")
     * True
     * >>> correct_bracketing("<<><>>")
     * True
     * >>> correct_bracketing("><<>")
     * False
     * @param brackets a string of "<" and ">".
     * @return true if every opening bracket has a corresponding closing bracket,false otherwise
     */
    public static Boolean correctBracketing(String brackets) {

File: Cleared_Dataset/HumanEval_Py/56_P13.py


def correct_bracketing(brackets: str):
    """ brackets is a string of "<" and ">".
    return True if every opening bracket has a corresponding closing bracket.

    args:
        brackets: string of "<" and ">"

    returns:
        True if every opening bracket has a corresponding closing bracket.

    examples:
    >>> correct_bracketing("<")
    False
    >>> correct_bracketing("<>")
    True
    >>> correct_bracketing("<<><>>")
    True
    >>> correct_bracketing("><<>")
    False
    """
